### PR TITLE
Add App folder, Create Login Page

### PR DESCRIPTION
**Pull request by bcain150**
This is just a PR to add a starting point for us to build from. I was able to fiddle around a bit and create a login page... unfortunately none of the buttons do anything. In order for them to work we will probably have to code a backend...

Also I got all of the login code from [this website tutorial](https://code.tutsplus.com/tutorials/common-react-native-app-layouts-login-page--cms-27639).

IMG_7304

### CODE REVIEW

**Comment by jjharmsy** [comment]
Is there any way to change the buttons from pink to purple?

File: README.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ This repo will be for the development of a bar app. There are infinite direction
         + `sardine-scale`: Amount of current attendees/how packed is it.
         + `music`: **type of music playing** (*live band?, DJ, performer*), **what genre** (*hiphop, rap, edm, country, rock, etc*). 
         + `age-range`: avg age of people in attendance
-        + `ragecage`: how intoxicated are people?
+        + `rage-gauge`: how intoxicated are people or how crazy is it.
         + `publicRating` vs `NiteLifeRating`: overall rating system for each bar, possible app generated rating system that takes into account a person's personal likes (*what we think you may like*)
-        + `drinkdeals`: section for promotions or drink deals ("*#lationight!, #bogoirishcarbombs!, etc*")
+        + `drink_deals`: section for promotions or drink deals ("*#lationight!, #bogoirishcarbombs!, etc*")
```
Comment (jjharmsy) [inline README.md:21]: if rage-gauge uses a hyphen i would use that here too instead of an underscore

**Review by jjharmsy — APPROVED** [review]

**Comment by bcain150** [comment]
I was gonna keep it a draft until I fix comments and I will also add a readme on how to run/replicate the homescreen.

File: README.md (hunk 1 above)
Comment (bcain150) [reply README.md:21]: I agree good catch.

**Comment by bcain150** [comment]
> Is there any way to change the buttons from pink to purple?

Yeah I can do that, I also think that this won't be the final thing. We can mess around with format, theme, and affects

**Pull request merged**